views/email/welcome.hbs

```diff
@@ -0,0 +1,17 @@
+{{!< email }}
+<h1>Firefox Monitor Report</h1>
+{{!-- TODO: get prettyDate helper in here --}}
```

Comment:
Not sure if this is a TODO for this PR, or for our future selves.

Reply:
Mostly future selves. And the hbs halpers and partials in nodemailer may take some refactoring.